### DISCUSSION TITLE
Extended properties of the google calendar events get erased

### DISCUSSION THREAD

**Discussion by NotGonnaGetMe** [kewisch/gdata-provider#736] [Support] [answered]
I am using the extended properties of the Google calendar events which I synchronize with Thunderbird. Everything is fine until I dismiss an event notification in Thunderbird, at that moment all of my extended properties are erased and replaced with X-MOZ-LASTACK private property. I assume that the plugin doesn't preserve other extended properties but this one. I hope this can be fixed

**Comment by kewisch** [comment, marked as answer]
Ugh yes, good point. I need to move to patching for other reasons, though I'm not sure it would help in this case. Given the architecture I don't really have a good place to roundtrip the other properties without exposing them in all exported ICS. Anyway, I've filed #747
